site/ja/tutorials/load_data/tf-records.ipynb

```diff
@@ -0,0 +1,1687 @@
+{
```

Comment:
This file name should be: `site/ja/tutorials/load_data/tf_records.ipynb`

Reply:
Thanks! I've just removed unnecessary one.